content/team/team.json

```diff
@@ -1,52 +1,176 @@
+          "discord": "Mike's Here#5684",
+          "email": " michael.liut@utoronto.ca",
+          "linkedin": "https://www.linkedin.com/in/mliut",
+          "website": "https://www.michaelliut.ca/"
+        }
+      ]
     },
     {
-      "name": "Nameera Azim",
-      "imgSrc": "/team/2021/Nameera.png",
-      "position": "Director of Communications",
-      "message": "Nameera is a third-year student specializing in Computer Science and minoring in both Statistics and Business. On lazy days, you can probably find her reading, painting, fangirling over an MCU movie, watching the Premier League, or messing around with her ukulele! Feel free to reach out!",
-      "discord": "Nameera#2802",
-      "email": "nameera.azim@mail.utoronto.ca",
-      "linkedin": "https://www.linkedin.com/in/nameera-azim-1924a21b6/"
+      "name": "Communications",
+      "members": [
+        {
+          "name": "Isha Kerpal",
+          "imgSrc": "/team/2021/Isha.jpg",
+          "position": "Communications Officer",
+          "message": "I enjoy handling the social media accounts for CSSC, helping plan our promotion strategies and responding to the questions we get on our platforms from students. I'm going to be a part of the Communications team at CSSC this year. I enjoy overseeing the planning of events and making sure the process is smooth for the entire CSSC team.",
+          "email": "isha.kerpal@mail.utoronto.ca",
+          "linkedin": "https://www.linkedin.com/in/isha-kerpal-6a923819b"
+        },
+        {
+          "name": "Leila Uy",
+          "imgSrc": "/team/2021/Leila.jpg",
+          "position": "Communications Officer",
+          "message": "In my free time, I enjoy going to unique food places with my friends. Whether it's taiyaki ice cream or 24-inch sliced pizza, I know a place we can go together and I'll try anything at least once.",
+          "discord": "KitKat#7118",
+          "email": "leila.uy@mail.utoronto.ca",
+          "linkedin": "https://www.linkedin.com/in/leila-uy-306272180"
+        }
+      ]
     },
     {
-      "name": "Naaz Sibia",
-      "imgSrc": "/team/2021/Naaz.png",
-      "position": "Director of Operations",
-      "message": "Naaz likes preference learning and Human-Computer Interaction (with a CS-ed focus). She also likes talking about mental health, and courses she's TA'ed. When not looking into the mentioned topics, or working to finish her bachelor's degree, she likes having 3 am discussions with friends, drawing chibis, and eating Chinese food.",
-      "discord": "Blossom#1234",
-      "email": "naaz.sibia@mail.utoronto.ca",
-      "linkedin": "https://www.linkedin.com/in/tuong-vy-n-614b4213b/",
-      "website": "http://naazsibia.com"
+      "name": "Logistics",
+      "members": [
+        {
+          "name": "Hayden Mak",
+          "imgSrc": "/team/2021/Hayden.jpg",
```

Comment:
We have some returning members like Hayden, Isha, Nitya, Jessica, Nameera. Can we change their past teams pic to use their more recent one and delete the old ones?

This also keeps the bundle size down (albeit marginally)